plotting/quicklook_generator.py

```diff
@@ -405,12 +405,148 @@ def ultra_general_rates_2_plot(self) -> None:
         raise NotImplementedError
 
 
+class SwapiQuicklookGenerator(QuicklookGenerator):
+    """SWAPI subclass for ULTRA quicklook plots."""
+
+    def two_dimensional_plot(self, variable: str = "") -> None:
+        """
+        Lead to correct function that will generate the desired quicklook plot.
+
+        Parameters
+        ----------
+        variable : str
+            Variable to specify which quicklook plot to generate.
+        """
+        match variable:
+            case "count rates":
+                self.swapi_count_rates()
+            case "absolute detection efficiency":
+                self.swapi_absolute_detection_efficiency()
+            case "count line":
+                self.swapi_count_line()
+
+    def swapi_count_rates(self) -> None:
+        """Generate SWAPI count rates plot."""
+        if self.data_set is None:
+            raise ValueError("Must load in a dataset.")
+
+        # Time data
+        x_values = self.data_set["epoch"].values
+        x_values_dt = convert_j2000_to_utc(x_values)
```

Comment:
```suggestion
        epoch_values = self.data_set["epoch"].values
        epoch_values_dt = convert_j2000_to_utc(epoch_values)
```

Nitpick here but I would prefer these be called "epoch" instead of "x", so at the end when you are calling the plotting method, it is easier to see what data are being plotted. Otherwise a reader of the code would have to look back and try to figure out what "x" is.